packages/api/src/resolvers/mutations/users/removePushSubscription.ts

```diff
@@ -0,0 +1,28 @@
+import { Context } from '@unchainedshop/types/api.js';
+import { log } from '@unchainedshop/logger';
+
+const removePushSubscription = async (_, __, context: Context): Promise<any> => {
+  const { modules, userId, userAgent } = context;
+  log(`mutation removePushSubscription ${userId}   ${userAgent}`);
+
+  try {
+    const isEnabled = await modules.users.pushEnabledByUser({ userId, userAgent });
+    if (isEnabled) throw new Error('Push Notification is not enabled by User');
```

Comment:
you shouldn't do that because the user could have multiple push subscriptions, the client needs to tell which subscription has to be removed the server cannot find out.